src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java

```diff
@@ -238,22 +238,28 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
         }
 
         MappingNode root = new MappingNode(Tag.MAP, tuples, BLOCK.getStyleBoolean());
+        serializeYamlNode(root, new OutputStreamWriter(res.getOutputStream(), StandardCharsets.UTF_8));
```

Comment:
Doing so we won't benefit the try-with-resource to close the stream.

Reply:
Silly me :) I'm fixing this